src/renderer.ts

```diff
@@ -1,3 +1,6 @@
 // This file is required by the index.html file and will
 // be executed in the renderer process for that window.
-// All of the Node.js APIs are available in this process.
+// No Node.js APIs are available in this process because
```

Comment:
// No Node.js APIs are available in this process because
// `nodeIntegration` is turned off. Use `preload.js` to
// selectively enable features needed in the rendering
// process.

let's instead say:
// No Node.js APIs are available in this process unless 
// `nodeIntegration` is set to `true` in `webPreferences`.
//  Use `preload.js` to selectively enable features
// needed in the renderer process.

Reply:
sounds good 👍

Reply:
updated the comment with https://github.com/electron/electron-quick-start-typescript/pull/38/commits/160e6e8a9dc9b3f5f1c15ec5ff7dc2e8b95da91e